### DISCUSSION TITLE
How to format Dates line in Adonis V4?

### DISCUSSION THREAD

**Discussion by kempsteven** [adonisjs/core#2179] [Help] [answered]
So I use this in adonis V4 for formatting dates:
https://adonisjs.com/docs/4.0/lucid#_casting_dates

```js
static get dates () {
    return super.dates.concat(['dob'])
}
  
static formatDates (field, value) {
    if (field === 'dob') {
      return value.format('YYYY-MM-DD')
    }
    return super.formatDates(field, value)
}
```
When using these in Adonis V5 im getting this errors:
``Property 'formatDates' does not exist on type 'LucidModel'`
`Property 'formatDates' does not exist on type 'LucidModel'`

Is there a way to do this in Adonis v5? I am getting milliseconds on my response, Thanks in advance!

**Comment by thetutlage** [comment, marked as answer]
You need to read the v5 docs for same. 

https://preview.adonisjs.com/guides/models/columns#columndate
https://preview.adonisjs.com/guides/models/serializing-models#customizing-value

**Reply by kempsteven** [reply]
I think I missed that page, Thankss!

**Reply by kempsteven** [reply]
![image](https://user-images.githubusercontent.com/51051704/105957677-89f75680-60b4-11eb-8353-575296f580e7.png)

Do I really have to do this, just to serialize every result?

**Reply by thetutlage** [reply]
Happy to explore alternatives, if you can suggest any 🙂
